docker/Dockerfile

```diff
@@ -0,0 +1,28 @@
+ARG PYTHON_DOCKER_TAG
+
+FROM python:${PYTHON_DOCKER_TAG}
+
+ARG GE_EXTRA_DEPS="spark,sqlalchemy,redshift,s3"
+
+ENV PYTHONIOENCODING utf-8
+ENV LANG C.UTF-8
+ENV HOME /home/great-expectations
```

Comment:
I'm totally open to pushback on this tiny nitpick: I enjoy shorter paths for the mounted `great_expectations` dir. Something like `WORKDIR /ge` then users only have to `docker run ... -v "$(pwd):/ge" ...`.

Reply:
I'm a little worried with permissions for directories at that level (so I never tried), could we use `/opt/ge` or `/home/ge` instead just to be safe? Happy to go ahead with `/ge` if you don't foresee any issues there (I just don't know enough about specifics of the different OSes)

Reply:
@Aylr I'm doing some DBT work in parallel, and came across this change in DBT https://github.com/fishtown-analytics/dbt/issues/2595, they are now using the `root` user and bring up good arguments for it. Should I change this image to use Root as well and use `/usr/app` ?

Reply:
1. I have not (yet) had problems using a root mounted directory, and it is much easier for users, but this isn't a hard requirement for me if it gives you heartburn.
2. Regarding the `root` user - it's definitely easier to use the root user for the reasons they state. While most best practice references do suggest using a non-privileged user, these are not in CLI/executable docker contexts but in typical service/server deployments inside docker. I think we're fine on this for now, and can increase the security in the future.

Reply:
I updated the Docker image to use Root, and tested it this morning on our Airflow/Kubernetes set up, it simplified our Airflow Operator code because I was able to remove an init-container which copied files around to fix permission issues